war/pom.xml

```diff
@@ -429,6 +429,7 @@ THE SOFTWARE.
         <artifactId>maven-license-plugin</artifactId>
         <!-- version specified in grandparent pom -->
         <configuration>
+          <!-- TODO does not correctly set resolved ${project.version} for jenkins-* artifacts in Incrementals mode -->
```

Comment:
Care to clarify? I opened jenkins-war at https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.119-rc15706.1e79797768ee/jenkins-war-2.119-rc15706.1e79797768ee.war and FWICT the value is correct:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<l:dependencies xmlns:l='licenses' groupId='org.jenkins-ci.main' artifactId='jenkins-war' version='2.119-rc15706.1e79797768ee'><l:dependency name='Jenkins war' groupId='org.jenkins-ci.main' artifactId='jenkins-war' version='2.119-rc15706.1e79797768ee' url='http://jenkins-ci.org/jenkins-parent/jenkins-war/'><l:description>Creates a war file. Also includes additional static web resources, such as images, CSS, JavaScript, and some HTML files.</l:description><l:license name='The MIT license' 
...
```